test/DynamoCoreWpfTests/ViewExtensions/DocumentationBrowserViewExtensionTests.cs

```diff
@@ -162,6 +165,9 @@ public void ViewExtensionIgnoresExternalEvents()
                 // Act
                 var tabsBeforeExternalEventTrigger = this.ViewModel.SideBarTabItems.Count;
                 viewExtension.HandleRequestOpenDocumentationLink(externalEvent);
+                DispatcherUtil.DoEventsLoop();
+
+                Assert.AreEqual(AsyncMethodState.NotStarted, viewExtension.BrowserView.initState);
```

Comment:
Why is it not started here? Won't webview2 initialization be complete by this point?

Reply:
using an externalLink will not startup webview2.
[That is what the test is verifying ](https://github.com/DynamoDS/Dynamo/blob/master/src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs#L471C29-L471C29)